config/consensus.go

```diff
@@ -137,8 +137,8 @@ type ConsensusParams struct {
 	DownCommitteeSize      uint64
 	DownCommitteeThreshold uint64
 
-	FilterTimeoutSmallLambdas             uint64
-	FilterTimeoutPeriod0SmallLambdas      uint64
+	AgreementFilterTimeout        time.Duration
+	AgreementFilterTimeoutPeriod0 time.Duration
```

Comment:
Add explanations around these variables - what do they mean, how they should be configured, etc.